lib/util/errorFileInfo.ts

```diff
@@ -25,6 +33,29 @@ const errorAddFileInfo = <T extends NodeError<T>>(error: T): void => {
       log.error(`ICE: can't parse title: ${title}`)
     }
   }
+  const errorEx = error as any as ErrorEx
+  const json = errorEx[jsonSymbol]
+  if (json !== undefined) {
+    const jsonInfo = getInfo(json)
+    if (jsonInfo !== undefined) {
+      const path = error.path
+      error.jsonPosition = getDescendantFilePosition(
+        json,
+        path === undefined ? undefined : isArray(path) ? path : path.split("/")
+      )
+      error.jsonUrl = getRootObjectInfo(jsonInfo).url
```

Comment:
>getRootObjectInfo(jsonInfo).url [](start = 22, length = 31)

is this url always going to be there or do we need to check ?

Reply:
It's optional so it could be `undefined`.